lib/gateway/index.js

```diff
@@ -49,11 +44,19 @@ module.exports = function ({ plugins, config } = {}) {
     });
 };
 
-function bootstrap({ plugins, config } = {}) {
-  const app = express();
-  app.set('x-powered-by', false);
+const bootstrapPolicies = ({ app, plugins, config } = {}) => {
```

Comment:
The `app` parameter does not seem to be used. Let's get rid of it.
```suggestion
const bootstrapPolicies = ({ plugins, config } = {}) => {
```

Reply:
`app` parameter is being used to register policy routes